parsers/ES_IB.py

```diff
@@ -87,17 +88,13 @@ def fetch_production(zone_key, session=None, target_datetime=None,
     island_data = fetch_island_data(zone_key, ses)
     data = []
 
-## biomass could probably be added as response.biomass in ree, including value['resid'] which I assume is for residuos=waste. Atm it seems unincluded.
-## I saw generation from "genAux" (generacion auxiliar) today on Formentera, which should probably be added to response.other in ree
-## Formentera mostly only has some solar generation during the day, importing from Ibiza all of the time, which probably has to be considered for response.production() > 0: ?
-
     if zone_key == 'ES-IB':
         expected_range = {'coal': (50,600)}
     else:
         expected_range = None
 
     for response in island_data:
-        if response.production() > 0:
+        if response.production() >= 0:
```

Comment:
I think we might have to actually remove this if statement - When I was looking at the ree site yesterday it was reporting small negative production values - my guess is onsite-consumption, but I am not sure - anyways, I think they are set to 0 if they are small enough in the validation steps later on (L120)

Reply:
I just saw your comment on the issue! Let's try this and see! Approved.